src/feed.js

```diff
@@ -11,12 +11,48 @@ import { CronJob } from "cron";
 import logger from "./logging";
 
 const transformFeedItem = (item) => {
+    const {
+        guid: id,
+        title: name,
+        contentSnippet: description,
+        isoDate: date_added,
+        author,
+        pubDate: publication_date,
+        content,
+        link,
+    } = item;
```

Comment:
The differences between destructuring syntax in Clojure and JavaScript had me confused for a few minutes. 😆

Reply:
I get confused by it, too. Especially since the variable renaming order is different in JS depending on if you're destructuring in the parameter definitions vs. in the body of the function.

Reply:
That was what confused me too.